clients/vault/src/metrics.rs

```diff
@@ -97,6 +99,8 @@ lazy_static! {
 		IntCounter::new("restart_count", "Number of service restarts")
 			.expect("Failed to create prometheus metric");
 }
+//[110, 97, 116, 105, 118, 101]
+const STELLAR_NATIVE_ASSET_TYPE: [u8; 6] = [110, 97, 116, 105, 118, 101];
```

Comment:
What does this do? What do the bytes represent?

Reply:
"native"